imgui-glow-renderer/src/lib.rs

```diff
@@ -3,6 +3,11 @@
 //! This is heavily influenced by the
 //! [example from upstream](https://github.com/ocornut/imgui/blob/fe245914114588f272b0924538fdd43f6c127a26/backends/imgui_impl_opengl3.cpp).
 //!
+//! It is important to note this renderer's API is not foolproof. It was designed
+//! more for simplicity and control (allowing the user to manually have some control
+//! over the GL state) than as a production-ready fully-general API. This control is
+//! why so many things are `pub`.
+//!
```

Comment:
I'd suggest removing this and instead extending the "Scope" section of the docs to something like this:

```
//! Consider this an example renderer. It is intended to be sufficent for simple
//! applications running imgui-rs as the final rendering step. If your application
//! has more specific needs, it's probably best to write your own renderer, in
//! which case this can be a useful starting point. This renderer is also not
//! foolproof (largely due to the global nature of the OpenGL state). For example,
//! a few "internal" funcitons are marked `pub` to allow the user more
//! fine-grained control at the cost of potential rendering errors.
```